src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AdditionalHearingDocumentServiceTest.java

```diff
@@ -54,7 +54,7 @@ public void setUp() {
         config.setAdditionalHearingFileName("AdditionalHearingDocument.pdf");
     }
 
-    @Test
+    @Test(expected = IllegalStateException.class)
```

Comment:
💣

Reply:
Just trying to get a green build for now, will remove before merge!